.jenkins/common.groovy

```diff
@@ -124,6 +125,10 @@ def runTestCommand (platform, project) {
             codeCovPackageOCL = "genhtml ocl-coverage-${platformOS}.info --output-directory coverage-${platformOS} && zip -r ocl-coverage-info-${platformOS}.zip coverage-${platformOS}"
             codeCovPackageHIP = "genhtml hip-coverage-${platformOS}.info --output-directory coverage-${platformOS} && zip -r hip-coverage-info-${platformOS}.zip coverage-${platformOS}"
         }
+        else {
+            nnTestsHIP = 'sudo python ../../tests/neural_network_tests/runNeuralNetworkTests.py --backend_type HIP'
```

Comment:
We don't have full support for the NN extension for the HIP backend yet. Running the NN tests may lead to some unexpected results.

Reply:
With this PR we can support MNIST with HIP, we can expand support going forward. This would be a good start. Attached is a report - http://math-ci.rocm.amd.com/job/compute-rocm-dkms-no-npi-hipclang/job/Short-GPUOpen/job/MIVisionX/job/PR-638/9/artifact/v7bPYvkUJ/mivisionx/build/release-opencl/inference_report_Linux-5.4.0-70-generic-x86_64-with-centos-8.1.1911-Core_20211004.md/*view*